Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/GoogleAPIVersion.kt

```diff
@@ -22,11 +22,15 @@ class GoogleAPIVersion @Inject constructor() {
             val currentVersion = InternalExposureNotificationClient.getVersion()
             currentVersion >= compareVersion
         } catch (apiException: ApiException) {
-            if (apiException.statusCode == CommonStatusCodes.API_NOT_CONNECTED) false
-            else throw apiException
+            isGoogleAPIConnected(apiException)
```

Comment:
@ralfgehrer  the catch block expects a boolean in return hence we need to return 'false'  here.

feel free to comment on it